catalog/core/catalog-core-standardframework/src/main/java/ddf/catalog/impl/operations/QueryOperations.java

```diff
@@ -713,6 +715,23 @@ protected Filter getNonVersionTagsFilter(Operation requestOperation) {
             filterBuilder.attribute(Metacard.TAGS).is().like().text(DeletedMetacard.DELETED_TAG)));
   }
 
+  private Filter getFilterWithValidationFilter() {
+    FilterBuilder builder = frameworkProperties.getFilterBuilder();
+    return builder.anyOf(
+        builder
+            .attribute(Validation.VALIDATION_ERRORS)
+            .is()
+            .like()
+            .text(FilterDelegate.WILDCARD_CHAR),
+        builder.attribute(Validation.VALIDATION_ERRORS).empty(),
```

Comment:
why do we want to include empty validation errors/warnings?

Reply:
this was an existed function that got moved here. the empty validation filter will filter out any record with error/warning. since the anyOf with WILDCARD, that will include all record with error/warning. It is a tautology